src/main/pages/che-7/installation-guide/examples/che-ref_supported-platforms-and-installation-methods.adoc

```diff
@@ -0,0 +1,102 @@
+// Module included in the following assemblies:
+//
+// supported_platforms
+
+[id="supported-platforms-and-installation-methods_{context}"]
+= Supported platforms and installation methods
+
+The following section provides information about the availability of {prod-short} {prod-ver} on Kubernetes and OpenShift infrastructures, and about their supported installation methods.
+
+== {prod} on Kubernetes
+
+{prod} can be installed on the Kubernetes infrastructures starting at version 1.9.
+
+The following options are described:
+
+* Running {prod-short} locally
+* Running {prod-short} in a public cloud provider
+* Running {prod-short} in a private cloud
+
+.Supported installation method for {prod-short} {prod-ver} on Kubernetes infrastructures when running {prod-short} locally
+[cols="2h,^"]
+|===
+|
+s| 1.9 or higher
```

Comment:
helm or operator for all cases when k8s infrastructure is run locally.

Reply:
This was changed by @l0rd